ui/static/ui/js/learning_resources.jsx

```diff
@@ -112,7 +116,9 @@ define('learning_resources', [
       if (this.props.selectedVocabulary.vocabulary_type === 'f') {
         allowTags = true;
       }
-      return <div className="col-sm-6">
+      return <div className="form-group">
+        <label htmlFor="{name}" className="col-sm-4 control-label">Terms</label>
```

Comment:
Same comment about `key` here. You can get rid of `key` here and above and replace with a `id` attribute which contains the vocab slug to prevent id collision.